packages/apollos-api/src/data/auth/index.js

```diff
@@ -3,6 +3,8 @@ import { Auth } from '@apollosproject/data-connector-rock';
 const { schema, resolver, contextMiddleware } = Auth;
 
 class dataSource extends Auth.dataSource {
+  expanded = true;
```

Comment:
could expanding Auth have unintended performance consequences?

Reply:
@conrad-vanl Potentially, yes. It's something I thought about when adding that attribute but haven't got metrics on yet. The alternative is to fetch a user's attributes within the `Campus` datasource itself, which will add another request but potentially reduce the time to resolve `currentUser`. 

The good news is that fetching the campus is faster overall (since we don't have to go through the `family` attribute). We also pretty much always get both the current user and their campus in willow, so if `expanded = true` is measurably slower, there are only a few places where we will be fetching that data needlessly. 

The bad news is that in those cases where we get the current user and don't want their campus (or get a user login, the only other call in `auth` that will be expanded) we potentially will pay that price. 

I would be happy to come up with a test to validate all of these assumptions, but after thinking it through last night I'm not particularly worried. I was thinking on my ride in about making `expanded` something you could define per request in core, rather than per dataSource, which might be the best long term solution.

Reply:
![image](https://user-images.githubusercontent.com/1637694/69354286-dabc2580-0c4d-11ea-9708-70bb48307d00.png)

Looks like `without loadAttributes` is twice as fast in my tests. 
![image](https://user-images.githubusercontent.com/1637694/69354314-e60f5100-0c4d-11ea-8f55-53d5e52454e9.png)